examples/coroutinesExample/src/main/java/io/realm/examples/coroutinesexample/di/DependencyGraph.kt

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.examples.coroutinesexample.di
+
+import com.dropbox.android.external.store4.Fetcher
+import com.dropbox.android.external.store4.SourceOfTruth
+import com.dropbox.android.external.store4.Store
+import com.dropbox.android.external.store4.StoreBuilder
+import io.realm.RealmConfiguration
+import io.realm.examples.coroutinesexample.data.newsreader.local.RealmNYTDao
+import io.realm.examples.coroutinesexample.data.newsreader.local.RealmNYTDaoImpl
+import io.realm.examples.coroutinesexample.data.newsreader.local.RealmNYTimesArticle
+import io.realm.examples.coroutinesexample.data.newsreader.local.insertArticles
+import io.realm.examples.coroutinesexample.data.newsreader.local.repository.NewsReaderRepository
+import io.realm.examples.coroutinesexample.data.newsreader.network.NYTimesApiClient
+import io.realm.examples.coroutinesexample.data.newsreader.network.NYTimesApiClientImpl
+import io.realm.examples.coroutinesexample.data.newsreader.network.model.NYTimesArticle
+import io.realm.examples.coroutinesexample.util.NewsReaderFlowFactory
+import kotlinx.coroutines.flow.map
+
+object DependencyGraph {
```

Comment:
Maybe just add a comment that this is our own DI solution for the sake of keeping the example simple?